<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>920</width>
    <height>700</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Character Selection</string>
  </property>
  <property name="styleSheet">
   <string notr="true">background-color: rgb(0, 0, 0);</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="assassin">
    <property name="geometry">
     <rect>
      <x>620</x>
      <y>0</y>
      <width>300</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>IMG/Blader.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="warrior">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>0</y>
      <width>300</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>IMG/Warrior.jpg</pixmap>
    </property>
   </widget>
   <widget class="QLabel" name="wizard">
    <property name="geometry">
     <rect>
      <x>310</x>
      <y>0</y>
      <width>300</width>
      <height>600</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>IMG/Wizard.jpg</pixmap>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_wiz_btn">
    <property name="geometry">
     <rect>
      <x>425</x>
      <y>625</y>
      <width>48</width>
      <height>49</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>IMG/Char_btn_wizard.gif</normaloff>IMG/Char_btn_wizard.gif</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>49</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_warrior_btn">
    <property name="geometry">
     <rect>
      <x>125</x>
      <y>625</y>
      <width>48</width>
      <height>49</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>IMG/Char_btn_warrior.gif</normaloff>IMG/Char_btn_warrior.gif</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>49</height>
     </size>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton_assassin_btn">
    <property name="geometry">
     <rect>
      <x>725</x>
      <y>625</y>
      <width>48</width>
      <height>49</height>
     </rect>
    </property>
    <property name="cursor">
     <cursorShape>PointingHandCursor</cursorShape>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="icon">
     <iconset>
      <normaloff>IMG/Char_btn_blader.gif</normaloff>IMG/Char_btn_blader.gif</iconset>
    </property>
    <property name="iconSize">
     <size>
      <width>48</width>
      <height>49</height>
     </size>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
